documentation/versioned_docs/version-1.x/migration-guide.md

```diff
@@ -143,3 +145,15 @@ where `PROJECT`, `MODELID`, `REVISIONID` and `APIKEY` must be set in the environ
 If the model isn't compatible with `@cognite/reveal`, it must be reprocessed. This can be done by [uploading a new revision in Cognite Data Fusion](https://docs.cognite.com/cdf/3d/guides/3dmodels_upload.html). When uploading a new revision, asset mappings must be recreated. If the node hierarchy in the new revision is identical to the previous model node IDs should be identical and the asset mappings can be copied from one revision to the other by using the [get3DMappings](https://docs.cognite.com/api/v1/#operation/get3DMappings) and [create3DMappings](https://docs.cognite.com/api/v1/#operation/create3DMappings) API endpoints.
 
 Another alternative is to use the experimental 'reprocess' endpoint which generates new model outputs for a 3D model and is less intrusive than uploading a new revision. Since this endpoint is experimental it's not publicly exposed in the API yet. Please contact [lars.moastuen@cognite.com](mailto:lars.moastuen@cognite.com) if you want to explore this option.
+
+## Common issues
+
+**`TypeError: e.applyMatrix4 is not a function`**
+
+If you experience  `TypeError: e.applyMatrix4 is not a function` after migrating from 
+`@cognite/3d-viewer` to `@cognite/reveal`, the problem is usually caused by missing or 
+outdated ThreeJS.
+
+To fix this issue, install or update ThreeJS using the 
+[`three`-package](https://www.npmjs.com/package/three). This should match the version 
+used by `@cognite/reveal`.
```

Comment:
```suggestion
```

I did exactly the same thing in my PR with threejs update. So just to avoid conflict pls merge the suggestion :) I also removed that info from next version docs because it shouldn't be the case.

Reply:
nvm for now as I converted PR to draft